app/src/main/res/layout/profile_progress_header.xml

```diff
@@ -12,36 +12,34 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
-
-    <FrameLayout
-      android:id="@+id/profile_frame_layout"
-      android:layout_width="96dp"
-      android:layout_height="96dp"
-      android:layout_marginTop="32dp"
+    android:layout_height="wrap_content"
+    android:clipToPadding="false"
+    android:paddingTop="32dp">
+
+    <com.jackandphantom.circularimageview.CircleImage
+      android:id="@+id/profile_edit_image"
+      android:layout_width="100dp"
+      android:layout_height="100dp"
+      android:clickable="true"
+      android:focusable="true"
+      android:onClick="@{() -> viewModel.clickOnProfilePicture()}"
+      app:add_shadow="true"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent">
-
-      <de.hdodenhof.circleimageview.CircleImageView
-        android:id="@+id/profile_edit_image"
-        android:layout_width="96dp"
-        android:clickable="true"
-        android:focusable="true"
-        android:onClick="@{() -> viewModel.clickOnProfilePicture()}"
-        android:layout_height="96dp"
-        profile:src="@{viewModel.profile.avatar}" />
-
-      <com.google.android.material.floatingactionbutton.FloatingActionButton
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="end|bottom"
-        android:src="@drawable/ic_edit_18"
-        app:backgroundTint="@color/white"
-        app:fabCustomSize="24dp"
-        android:onClick="@{() -> viewModel.clickOnProfilePicture()}"
-        app:tint="@color/colorPrimary" />
-    </FrameLayout>
+      app:layout_constraintTop_toTopOf="parent"
+      app:shadow_radius="8"
+      profile:src="@{viewModel.profile.avatar}" />
+
+    <ImageView
+      android:layout_width="48dp"
+      android:layout_height="48dp"
+      android:onClick="@{() -> viewModel.clickOnProfilePicture()}"
+      android:paddingStart="10dp"
+      android:paddingTop="10dp"
```

Comment:
Can this be in multiple of `4` or is it not possible here?

Reply:
Done